modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './index.scss';
+
+import {useState} from 'react';
+
+import NavigationBar from '../../../../../common/components/navigation-bar';
+import {getWebDavUrl} from '../../../../../common/utils/webdav';
+
+const PolicyDetail = () => {
+	const amountOfPoliceDetail = [0];
+	const navbarLabel = ['Vehicles', 'Drives'];
+	const [active, setActive] = useState(navbarLabel[0]);
+
+	return (
+		<div className="policy-detail-container">
+			<div className="bg-neutral-0 policy-detail-title pt-3 px-5 rounded-top">
+				<h5 className="m-0">Policy Detail</h5>
+			</div>
+
+			<div className="d-flex flex-row">
+				<NavigationBar
+					active={active}
+					navbarLabel={navbarLabel}
+					setActive={setActive}
+				/>
+			</div>
+
+			{amountOfPoliceDetail.map(
+				(curentPoliceDetail, indexPoliceDetail) => (
```

Comment:
Please, put the type of parameters here.
```suggestion
				(curentPoliceDetail:?, indexPoliceDetail:number) => (
```

Reply:
Done!